deploy/charts/cert-manager/values.yaml

```diff
@@ -28,7 +28,7 @@ global:
     enabled: false
     useAppArmor: true
 
-  # Set the verbosity of cert-manager. Range of 0 - 6 with 6 being the most verbose.
+  # Set the verbosity of cert-manager. Range of 0 - 5 with 5 being the most verbose.
```

Comment:
We can't set '5' as the most verbose- although that is the most verbose setting used in cert-manager code, we pass that log level into kube libraries that we use where different log levels are used. Is there a way to do this so that the upper cap is not enforced?

Reply:
see https://github.com/cert-manager/cert-manager/pull/5860#issuecomment-1875103608